docs/README.md

```diff
@@ -1,28 +1,116 @@
-# README
+# Documentation for Unity Catalog
 
-## Serving the documentation with mkdocs
+Unity Catalog uses [**MkDocs**](https://www.mkdocs.org/) to manage and serve its documentation. MkDocs is a simple, static site generator that’s geared towards project documentation and is written in Python. This guide will help you set up the local environment, serve the documentation locally, and deploy it to GitHub Pages.
```

Comment:
Personal opinion: Probably add a line break in between (around character 120). Applies to all longer lines.